packages/flutter/lib/src/painting/box_painter.dart

```diff
@@ -1127,6 +1115,114 @@ enum BoxShape {
   circle
 }
 
+/// An immutable set of radii for each corner of a rectangle. Only used with
+/// [BoxShape.rectangle].
+class BorderRadius {
+  /// Creates a border radius where all radii are [radius].
+  const BorderRadius.all(Radius radius) : this.only(
+    topLeft: radius,
+    topRight: radius,
+    bottomRight: radius,
+    bottomLeft: radius
+  );
+
+  /// Creates a border radius where all radii are [Radius.circular(radius)].
+  BorderRadius.allCircular(double radius) : this.all(
```

Comment:
This name makes sense, and the "all" qualifier nicely ties it to BorderRadius.all(). On the other hand it's going to be used a great deal and the other constructors are unqualified. In other words: consider calling it BorderRadius.circular().